docs/Speech and Audio/Playing Spoken Feedback/index.md

```diff
@@ -6,16 +6,55 @@ When using the @![iOS]`SDLSpeak`!@@![android,javaSE,javaEE,javascript]`Speak`!@
 ## Creating the Speak Request
 The speech request you send can simply be a text phrase, which will be played back in accordance with the user's current language settings, or it can consist of phoneme specifications to direct SDL’s TTS engine to speak a language-independent, speech-sculpted phrase. It is also possible to play a pre-recorded sound file (such as an MP3) using the speech request. For more information on how to play a sound file please refer to [Playing Audio Indications](Speech and Audio/Playing Audio Indications). 
 
-### Getting Head Unit Speech Capabilities
-To get the head unit's supported speech capabilities, check the @![iOS]`SDLSystemCapabilityManager.speechCapabilities`!@@![android,javaSE,javaEE,javascript] `sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.SPEECH)`!@ after successfully connecting to the head unit. Below is a list of commonly supported speech capabilities.
+### Getting the Supported Speech Capabilities
+Once you have successfully connected to the module, you can access supported speech capabilities properties on the @![iOS]`SDLManager.systemCapabilityManager`!@@![android,javaSE,javaEE,javascript]`sdlManager.getSystemCapabilityManager()`!@ instance.
+
+@![iOS]
+##### Objective-C
+```objc
+NSArray<SDLSpeechCapabilities> *speechCapabilities = self.sdlManager.systemCapabilityManager.speechCapabilities;
```

Comment:
```suggestion
NSArray<SDLSpeechCapabilities *> *speechCapabilities = self.sdlManager.systemCapabilityManager.speechCapabilities;
```

I think that's needed?

Reply:
I get a compile error for: `NSArray<SDLSpeechCapabilities *> *speechCapabilities`

Reply:
You're correct, `SDLSpeechCapabilities` is an enum.